.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thank you for contributing to our devenv project!
+
+It would be helpful, if you gave us as much information as you could to process your pull request better.
```

Comment:
```suggestion
It would be helpful if you could provide us with as much information as possible so we can better process your pull request.
```

Reply:
Ich würde das `with` noch weglassen wollen. Gibt dem eine etwas andere Bedeutung, finde ich.

Reply:
To me, it sounds more appropriate with the "with". But both fit well for me

Reply:
Okay, then let's just take it. ^^